cmd/juju/controller/showcontroller.go

```diff
@@ -141,8 +145,12 @@ func (c *showControllerCommand) Run(ctx *cmd.Context) error {
 			continue
 		}
 		modelTags := make([]names.ModelTag, len(allModels))
+		var controllerModelUUID string
 		for i, m := range allModels {
 			modelTags[i] = names.NewModelTag(m.UUID)
+			if m.Name == bootstrap.ControllerModelName {
```

Comment:
So is this just because from the client side we don't know the controller UUID?
should we be opening a bug that we should be returning that information earlier so that we don't have to infer it from the bootstrap name?